docs/contributors/maintainers.md

```diff
@@ -76,12 +78,16 @@ ___
 
 - Open [github releases](https://github.com/process-analytics/bpmn-visualization-js/releases)
 - Rename the existing `In Progress` draft release (updated before starting the release) to match the version that has just been tagged
-- Assign the new tag as release target and save the draft (this may have been already managed by `release-drafter`)
 - Ensure that `This is a pre-release` is unchecked (except if we are releasing alpha, beta, rc, ...)
 - In the release description (check previous releases as a source of inspiration)
   - at least add/update a link to the related milestone
   - put screenshots/gif of the new features
-- Published the release **only when you are done** with the release content. At any time, you can save the draft
+- If an old `Notes for next release notes` exists, use its content and delete this release
```

Comment:
Following the action to do at each release, we should always have a draft release note named `Notes for next release notes`.

Reply:
Yes that's true. But we cannot reuse it, otherwise after a few release it will be place far away of the next release. What we want is to have this draft release on top of the list to easily edit it during development.

Reply:
No problem ^^